eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/CreateTopicProcessor.java

```diff
@@ -45,9 +46,10 @@
 
 import com.fasterxml.jackson.core.type.TypeReference;
 
+@EventMeshTrace(isEnable = true)
```

Comment:
This processor is used to manage metadata. I think trace only needs to be used on the link of event sending and receiving.